src/main/resources/org/sagebionetworks/web/client/widget/entity/renderer/EntityListWidgetViewImpl.ui.xml

```diff
@@ -26,7 +26,7 @@
 						<t:TableHeader width="165px" addStyleNames="visible-md visible-lg">
 							<bh:Span text="Date" />
 						</t:TableHeader>
-						<t:TableHeader width="310px" addStyleNames="visible-xl">
+						<t:TableHeader width="310px" addStyleNames="visible-md visible-lg">
```

Comment:
I believe this will hide the column on xl devices (please see the table in the CSS responsive layout utils section of the bootstrap docs).